config/memory_persister.go

```diff
@@ -0,0 +1,49 @@
+// Licensed under the Apache License, Version 2.0
+// Details: https://raw.githubusercontent.com/maniksurtani/quotaservice/master/LICENSE
+
+package config
+
+import (
+	"errors"
+	"io"
+)
+
+type MemoryConfigPersister struct {
+	config  io.Reader
```

Comment:
It seemed "cleaner" to do Reader in and out, but bytes, or even actual ServiceConfig structs, would work.